open-api/rest-catalog-open-api.yaml

```diff
@@ -217,12 +217,14 @@ paths:
       parameters:
         - $ref: '#/components/parameters/page-token'
         - $ref: '#/components/parameters/page-size'
+        - $ref: '#/components/parameters/delimiter'
         - name: parent
           in: query
           description:
             An optional namespace, underneath which to list namespaces.
             If not provided or empty, all top-level namespaces should be listed.
-            If parent is a multipart namespace, the parts must be separated by the unit separator (`0x1F`) byte.
+            If parent is a multipart namespace, the parts must be separated by the namespace separator as
+            indicated via the `delim` query parameter, which defaults to the unit separator (`0x1F`) byte.
```

Comment:
I'd propose softer language for backward compatibility with older clients (since the API version is not increased)... Something like "Clients are encouraged to use the delimiter advertised by the server (#10877). However, clients can choose another delimiter, in which case they must set the `delim` query parameter.". If `delim` is not set servers should decode namespaces as defined in #10877 (I guess we'll have to update the spec text depending on which PR merges first).